Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml

```
@@ -179,6 +187,10 @@
     sprite: Objects/Consumable/Smokeables/Cigarettes/Packs/black.rsi
   - type: Item
     sprite: Objects/Consumable/Smokeables/Cigarettes/Packs/black.rsi
+  - type: StorageFill
```

Ну и здесь уже, раз речь пошла
```suggestion
  - type: StorageFill
```